rclcpp/include/rclcpp/node_builtin_executor.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Sony Group Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLCPP__NODE_BUILTIN_EXECUTOR_HPP_
+#define RCLCPP__NODE_BUILTIN_EXECUTOR_HPP_
+
+#include <memory>
+
+#include "rclcpp/executor.hpp"
+#include "rclcpp/macros.hpp"
+#include "rclcpp/node_interfaces/node_base_interface.hpp"
+#include "rclcpp/node_interfaces/node_logging_interface.hpp"
```

Comment:
Please remove `node_logging_interface` and `node_parameters_interface.hpp`.

Reply:
Okay